Dockerfile

```diff
@@ -0,0 +1,113 @@
+FROM ubuntu:18.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get -y update \
+    && apt-get -y install --no-install-recommends apt-utils dialog 2>&1 \
+    # stuff
+    && apt-get -y install git npm less iproute2 \
+    build-essential \
+    gcc-5 g++-5 libtool cmake curl debconf-utils \
+    git-core liblua5.1 \
+    liblua5.1-dev libluabind-dev  \
+    minizip make locales \
+    nano unzip uuid-dev iputils-ping \
+    zlibc wget \
+    gdb valgrind \
+    # vita sdk 
+    python sudo autoconf texinfo bison flex \
+    # wii
+    gdebi-core ca-certificates pkg-config bzip2 xz-utils bsdtar doxygen gnupg \
+    # sdl2
+    libsdl2-dev \
+    libsdl2-image-dev libsdl2-mixer-dev libsdl2-ttf-dev \
+    # windows support
+    mingw-w64 \
+    # xbox
+    clang lld llvm llvm-dev \
+    # raylib
+    libasound2-dev mesa-common-dev libx11-dev libxrandr-dev libxi-dev xorg-dev libgl1-mesa-dev libglu1-mesa-dev \
+    #clean up
+    && apt-get autoremove -y \
+    && apt-get clean -y \
+    && rm -rf /var/lib/apt/lists/*
+
+# for sdl2 in windows, we have to get the mingw source:
+RUN wget https://www.libsdl.org/release/SDL2-devel-2.0.12-mingw.tar.gz -O /tmp/mingw.tar.gz
+RUN cd /tmp \
+    && tar xzf mingw.tar.gz \
+    && cp -R SDL2-2.0.12/x86_64-w64-mingw32/include/SDL2 /usr/x86_64-w64-mingw32/include/SDL2 \
+    && cp -R SDL2-2.0.12/x86_64-w64-mingw32/lib/* /usr/x86_64-w64-mingw32/lib/ \
+    && cp SDL2-2.0.12/x86_64-w64-mingw32/bin/* /usr/x86_64-w64-mingw32/bin/ \
+    && rm -rf /tmp/*
+RUN cd /tmp \
+    && wget https://github.com/raysan5/raylib/releases/download/3.0.0/raylib-3.0.0-Linux-amd64.tar.gz -O /tmp/raylib.tar.gz \
+    && tar xzf raylib.tar.gz \
+    && cp -R raylib-3.0.0-Linux-amd64/include/raylib.h /usr/x86_64-w64-mingw32/include/raylib.h \
+    && cp -R raylib-3.0.0-Linux-amd64/lib/* /usr/x86_64-w64-mingw32/lib/ \
+    && rm -rf /tmp/*
+
+# vita SDK
+ENV VITASDK /usr/local/vitasdk
+ENV PATH=$VITASDK/bin:$PATH
+RUN cd /tmp \
+    && git clone https://github.com/vitasdk/vdpm \
+    && cd vdpm \
+    && ./bootstrap-vitasdk.sh \
+    && ./install-all.sh \
+    && rm -rf /tmp/*
+
+# wii, nds, gamecube, devkit pro
+RUN cd /tmp \
+    && wget https://github.com/devkitPro/pacman/releases/latest/download/devkitpro-pacman.amd64.deb \
+    && gdebi -n devkitpro-pacman.amd64.deb \
+    && rm devkitpro-pacman.amd64.deb \
+    && dkp-pacman -Scc --noconfirm
+RUN dkp-pacman -Syyu --noconfirm gamecube-dev wii-dev wiiu-dev nds-dev \
```

Comment:
When using this on Docker-for-Windows, I am getting the following error `error: could not open file: /etc/mtab: No such file or directory`:
![grafik](https://user-images.githubusercontent.com/1042579/134805466-aed20b17-dec4-4de2-b1d6-c70021f9b078.png)

This can be resolved by adding `&& sudo ln -s /proc/self/mounts /etc/mtab \` after line 63